### DISCUSSION TITLE
Can't receive a verification code

### DISCUSSION THREAD

**Discussion by Mridul-DPA** [NativeScript/NativeScript#9949] [General]
Hello there,

I am trying to verify my phone number to ask support via "NativeScript discord" but I couldn't receive the verification code for that (tried many times with two different numbers). Could anyone please check that?

Thanks in advance!

**Comment by rigor789** [comment]
There's nothing on our side we can do, this is what I found on the Discord support site
 https://support.discord.com/hc/en-us/articles/360000961212-Phone-Verification#:~:text=If%20you%20receive%20the%20Invalid,out%20the%20text%20entry%20field.
